src/client/app/containers/CompareChartContainer.ts

```diff
@@ -30,21 +44,43 @@ function mapStateToProps(state, ownProps) {
 	// Compose the text to display to the user.
 	let delta;
 	if (ownProps.isGroup) {
-		delta = change => {
-			if (isNaN(change)) return '';
-			if (change < 0) return `${state.groups.byGroupID[ownProps.id].name} has used ${parseInt(change.toFixed(2).replace('.', '').slice(1))}% less energy this week.`;
-			return `${state.groups.byGroupID[ownProps.id].name} has used ${parseInt(change.toFixed(2).replace('.', ''))}% more energy this week.`;
+		delta = (change: number) => {
```

Comment:
The logic here is almost identical for both meters and groups. Can you merge these together?

Reply:
No. We have already had this discussion, when I was working on group graphing.